test/integration/testHTAP.js

```diff
@@ -80,6 +80,9 @@ if (process.env.CLOUD_PROVIDER === 'AWS') {
               connection.execute({
                 sqlText: sqlTexts[k],
                 complete: function (err) {
+                  if (err) {
+                    console.log(err)
```

Comment:
I believe linter is right here :)

Reply:
fully agree and this is not part of the original PR or my intention and will be removed from the final version
i really just wanted to see why the HTAP tests break and if them breaking is anyhow related to my change. (it is not, 'create hybrid table' fails with Snowflake internal crash, which I don't think is related to this client side logging PR)

Reply:
and reverted